zkevm-circuits/src/state_circuit/random_linear_combination.rs

```diff
@@ -38,6 +38,7 @@ impl<const N: usize> Config<N> {
     ) -> Result<(), Error> {
         let bytes = value.to_le_bytes();
         for (i, &byte) in bytes.iter().enumerate() {
+            region.name_column(|| format!("STATE_RLC_byte[{}]", i), self.bytes[i]);
```

Comment:
Same comment to follow approach in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1167/files

Reply:
fixed in [1b5aa41](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1152/commits/1b5aa4193380414a8378031399be7a6eddfac11a)